### PR TITLE
Add Card component

### PR DESCRIPTION
**Pull request by voxtex**
My first shot at contributing, let me know if you see anything I should change.

### CODE REVIEW

File: src/Antd_Card.re [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        ~type_?,
+        ~onTabChange?,
+        (),
+      ),
+    children,
+  );
+
+module Grid = {
+  [@bs.module "antd/lib/card"]
+  external reactClass : ReasonReact.reactClass = "Grid";
```
Comment (jthegedus) [inline src/Antd_Card.re]: is this meant to be Card?

**Comment by Enalmada** [comment]
Card renders fine for me during initial testing.  I did have to add style to the card parameters though.  Thank you very much for taking the time to contribute this.  Have any others, even if they are not perfect?  I am trying to collect what is out there to figure out what still needs doing. 

Note that there are bindings for components I need in my fork and will do pull request when tested: https://github.com/Enalmada/bs-ant-design

**Comment by Enalmada** [comment]
@voxtex Can you take a look at this branch and let me know if there is anything missing that you use and care about? https://github.com/Enalmada/bs-ant-design.

**Comment by voxtex** [comment]
@Enalmada sorry to say I no longer actively use ReasonML, but it looks good as far as I can tell. Too bad my PR was never looked at or merged.

**Pull request closed without merging**